yb-voyager/cmd/common.go

```diff
@@ -497,9 +503,49 @@ func initMetaDB(migrationExportDir string) *metadb.MetaDB {
 		utils.ErrExit("could not init migration status record: %w", err)
 	}
 
+	// generate salt and initialise the anonymiser
+	salt, err := loadOrGenerateAnonymisationSalt(metaDBInstance)
+	if err != nil {
+		utils.ErrExit("could not load or generate anonymisation salt: %v", err)
+	}
+
+	anonymizer, err = anon.NewVoyagerAnonymizer(salt)
```

Comment:
It's a little weird that we're initializing the anonymizer inside a function that's called `initMetaDB`? can we have a separate function and call it specifically wherever required? 

I imagine it will just be a call from the root command so it's only one place where we'll have to make an extra func call.

Reply:
@makalaaneesh 
There are two paths:
1. MetaDB gets created/initialised when export-dir structure is created - `CreateMigrationProjectIfNotExists`

2. But `CreateMigrationProjectIfNotExists` is not always called, for eg: import data expected the project to be there. So in those case we check if metaDB is already created and just initialise it.

The code needs some refactoring and simplification.
Will add a TODO comment for now, otherwise this anonymizer initialisation code will also be scatter at 2-3 different places.